File: inception/inception-api-annotation-experimental/src/main/java/de/tudarmstadt/ukp/inception/experimental/api/messages/request/SelectAnnotationRequest.java
package de.tudarmstadt.ukp.inception.experimental.api.messages.request;

import de.tudarmstadt.ukp.clarin.webanno.api.annotation.model.VID;

public class SelectAnnotationRequest
{
    private String clientName;
    private String userName;
    private long projectId;
    private long documentId;
    private VID annotationAddress;

    public String getClientName()
    {
        return clientName;
    }

    public void setClientName(String aClientName)
    {
        clientName = aClientName;
    }

    public String getUserName()
    {
        return userName;
    }

    public void setUserName(String aUserName)
    {
        userName = aUserName;
    }

    public long getProjectId()
    {
        return projectId;
    }

    public void setProjectId(long aProjectId)
    {
        projectId = aProjectId;
    }

    public long getDocumentId()
    {
        return documentId;
    }

    public void setDocumentId(long aDocumentId)
    {
        documentId = aDocumentId;
    }

    public VID getAnnotationAddress()
    {
        return annotationAddress;
    }

    public void setAnnotationAddress(VID aAnnotationAddress) {
        annotationAddress = aAnnotationAddress;
    }
}

File: inception/inception-api-annotation-experimental/src/main/ts/client/messages/request/SelectAnnotationRequest.ts
export class SelectAnnotationRequest
{
    clientName : string;
    userName : string;
    projectId : number;
    documentId : number;
    annotationAddress : number;

    constructor(aClientName: string, aUserName: string, aProjectId: number, aDocumentId: number, aAnnotationAddress: number)
    {
        this.clientName = aClientName;
        this.userName = aUserName;
        this.projectId = aProjectId;
        this.documentId = aDocumentId;
        this.annotationAddress = aAnnotationAddress;
    }
}